docs/views/reference/filters.jade

```diff
@@ -4,8 +4,11 @@ block documentation
   h2 Filters
 
   p.
-    Filters let you use other languages within a jade template.
+    Filters let you use other languages within a jade template. 
     They take a block of plain text as an input.
+  p.
+    All #[a(href='//www.npmjs.com/browse/keyword/jstransformer' target='_blank') JSTransformers] can be used as jade filters. 
+    Popular filters include #[code :coffee-script], #[code :babel], #[code :uglify-js], #[code :less], and #[code :markdown].
```

Comment:
We should encourage `markdown-it` instead of `markdown`.  The `markdown` package on npm is not being actively maintained.